addons/crm/migrations/13.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,89 @@
+---Models in module 'crm'---
+new model crm.lead.scoring.frequency
+new model crm.lead.scoring.frequency.field
+# NOTHING TO DO
+
+---Fields in module 'crm'---
+crm          / crm.lead                 / automated_probability (float) : NEW
+crm          / crm.lead                 / email_cc (text)               : type is now 'char' ('text')
+crm          / crm.lead                 / email_normalized (char)       : NEW isfunction: function, stored
+crm          / crm.lead                 / email_state (selection)       : NEW selection_keys: ['correct', 'incorrect'], isfunction: function, stored
+crm          / crm.lead                 / lang_id (many2one)            : NEW relation: res.lang
+crm          / crm.lead                 / phone_state (selection)       : NEW selection_keys: ['correct', 'incorrect'], isfunction: function, stored
+# NOTHING TO DO
+crm          / crm.lead.scoring.frequency / lost_count (float)            : NEW
+crm          / crm.lead.scoring.frequency / team_id (many2one)            : NEW relation: crm.team
+crm          / crm.lead.scoring.frequency / value (char)                  : NEW
+crm          / crm.lead.scoring.frequency / variable (char)               : NEW
+crm          / crm.lead.scoring.frequency / won_count (float)             : NEW
+crm          / crm.lead.scoring.frequency.field / field_id (many2one)           : NEW relation: ir.model.fields, required
+# NOTHING TO DO: New models
+crm          / crm.stage                / is_won (boolean)              : NEW
```

Comment:
Put is_won next to probability. In pre-migration: copy-column of probability using rename to None. In post-migration: set "is_won" = True if "stage.probability" = 100

Reply:
Thanks. changed